.travis.yml

```diff
@@ -1,15 +1,56 @@
+---
+sudo: required
+services:
+  - docker
 language: python
-
+python:
+  - "2.7"
+  - "3.4"
+env:
+  - OS_NAME=centos OS_VERSION=6
+  - OS_NAME=centos OS_VERSION=7
+  - OS_NAME=ubuntu OS_VERSION=trusty
+  - OS_NAME=ubuntu OS_VERSION=xenial
+before_install:
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt update
+  - sudo apt install -y docker-ce
+  - docker --version
 install:
-    - pip install tox
-
+  - wget http://archive.ubuntu.com/ubuntu/pool/universe/r/rpmlint/rpmlint_1.7-1_all.deb
+  - sudo dpkg -i rpmlint_1.7-1_all.deb || sudo apt install -f -y
+  - gem install mdl
+  - pip install tox
 script:
-    - tox
-
-env:
-    - TOXENV=py27
-    - TOXENV=py34
-    - TOXENV=pep8
-    - TOXENV=coveralls
-
-sudo: false
+  # First syntax, linting and pre-test stages
+  - mdl -s relaxed README.md
+  - rpmlint rpm/
+  - tox -e pep8,bandit
+  # Then unit tests
+  - if [[  $TRAVIS_PYTHON_VERSION == 2.7 ]]; then tox -e py27; fi
```

Comment:
there's gotta be a way to write these two if's in a single line, using the environment variables at disposal.

Reply:
Any more precise suggestion?